amundsen_application/static/templates/index.html

```diff
@@ -1,15 +1,9 @@
 <html>
   <head>
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=<%= htmlWebpackPlugin.options.config.google.key%>"></script>
-    <script>
-      const google = <%= JSON.stringify(htmlWebpackPlugin.options.config.google) %>;
-      window.dataLayer = window.dataLayer || [];
-      function gtag() { dataLayer.push(arguments); }
-      gtag('js', new Date());
-      gtag('config', google.key);
-      gtag('create', google.key, { 'siteSpeedSampleRate': google.sampleRate });
-    </script>
+
+{% if <%= htmlWebpackPlugin.options.config.google.enabled%> %}
+{% include 'fragments/google-analytics-loader.html' %}
+{% endif %}
```

Comment:
@ttannis any idea why this causes the following Python FE test to fail? I'm at a loss.

https://github.com/lyft/amundsenfrontendlibrary/blob/a5368433b44220982726608483c54fb7b7607e7c/tests/unit/api/search/test_v0.py#L164-L177

Reply:
It looks like something in this API causes Flask to parse and render `index.html`, which has obviously bad syntax because it hasn't been processed yet while running tests. But what triggers this?

Reply:
My only idea now is that some of the tests require creating a an application instance because we cannot test certain things outside of the application context. What happens if we tweak the following in some way:
https://github.com/lyft/amundsenfrontendlibrary/blob/a5368433b44220982726608483c54fb7b7607e7c/tests/unit/api/search/test_v0.py#L10

Perhaps removes the use of 'static/templates'? I see the metadata tests work fine with just the `TestConfig`

Reply:
Removed the reference to `static/templates` and fixed the tests.

Reply:
Nvm it actually broke more tests. Locally it was defaulting to `static/dist/templates` which I had created from a previous build but those don't exist on the test framework.